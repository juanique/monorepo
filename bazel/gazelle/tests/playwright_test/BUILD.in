load("//bazel/playwright:defs.bzl", "playwright_test")

# This file intentionally empty - Gazelle should generate the appropriate rule
