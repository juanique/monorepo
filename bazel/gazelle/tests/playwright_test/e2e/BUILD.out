load("//bazel/js:defs.bzl", "playwright_test")

playwright_test(
    name = "e2e_test",
    srcs = [
        "example.pw.spec.js",
        "example.pw.spec.ts",
    ],
    deps = ["//:node_modules/@playwright/test"],
)
